lib/cli/invocation/git/runOnGitHook.ts

```diff
@@ -14,17 +14,40 @@
  * limitations under the License.
  */
 
-import { logger } from "@atomist/sdm";
+import {
+    configureLogging,
+    logger,
+    LoggingConfiguration,
+    NoLogging,
+} from "@atomist/automation-client";
 import { DefaultWorkspaceContextResolver } from "../../../common/binding/defaultWorkspaceContextResolver";
 import { isAtomistTemporaryBranch } from "../../../sdm/binding/project/FileSystemProjectLoader";
 import { AutomationClientInfo } from "../../AutomationClientInfo";
 import { argsToGitHookInvocation } from "../../entry/argsToGitHookInvocation";
-import { infoMessage, logExceptionsToConsole } from "../../ui/consoleOutput";
+import {
+    infoMessage,
+    logExceptionsToConsole,
+} from "../../ui/consoleOutput";
 import { renderEventDispatch } from "../../ui/renderClientInfo";
 import { suggestStartingAllMessagesListener } from "../command/support/suggestStartingAllMessagesListener";
 import { AutomationClientFinder } from "../http/AutomationClientFinder";
 import { defaultAutomationClientFinder } from "../http/support/defaultAutomationClientFinder";
-import { GitHookInvocation, handleGitHookEvent } from "./handleGitHookEvent";
+import {
+    GitHookInvocation,
+    handleGitHookEvent,
+} from "./handleGitHookEvent";
+
+const loggingConfiguration: LoggingConfiguration = {
+    console: {
+        ...NoLogging.console,
+        level: "warn",
```

Comment:
This should also be enabled if you want to actually output warn and above? Or perhaps use PlainLogging.